OpenUpSA/wazimap-ng-ui
# introducing line charts

## Description
displaying the profile indicator data as a line chart. 

documentation : 
https://app.gitbook.com/o/-Ly4_0FbvQFY-m3TOzuc/s/-M3zapxOsLjq_qKRcmH8/profile-indicator-configuration#setting-the-chart-type

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-406

## How is it tested automatically?

## How should a reviewer test it locally
* in the admin panel set the `chartType`
* confirm that the chart is displayed according to the type you selected
* confirm that line chart is displaying the data correctly by comparing it with the bar chart
* confirm that the downloading a line chart is working the same way as downloading a bar chart
* confirm that the chart is displayed in values and not in percentages(no matter what the chart config is)
* confirm that on the UI it is not possible to switch to percentages

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/173012802-d6a64054-4846-4c0b-874a-53b193e24d1a.png)


## Changelog

### Added
* `src/js/profile/charts/lineChart.js` to display a line chart

### Updated
* `src/js/profile/chart.js` to decide whether to display the data as a line chart or a bar chart

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
